files/jquery.quicksearch/update.json

```diff
@@ -0,0 +1,10 @@
+
+{
+  "packageManager": "github",
+  "name": "jquery.quicksearch",
+  "repo": "DeuxHuitHuit/quicksearch",
+  "basepath": "dist/",
```

Comment:
This is wrong, please use the documentation https://github.com/jsdelivr/libgrabber